sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -702,7 +702,69 @@ public static BinaryData fromFile(Path file) {
      * @throws UncheckedIOException if the file does not exist.
      */
     public static BinaryData fromFile(Path file, int chunkSize) {
-        return new BinaryData(new FileContent(file, chunkSize));
+        return new BinaryData(new FileContent(file, chunkSize, null, null));
+    }
+
+    /**
+     * Creates a {@link BinaryData} that uses the content of the file at {@link Path file} as its data. This method
```

Comment:
Should we mention that this overload defaults the chunkSize to STREAM_READ_SIZE?